components/LifeAtTorchbox/ValuesCarousel/ValuesCarousel.module.scss

```diff
@@ -0,0 +1,88 @@
+.container {
+    padding: $grid;
+    display: flex;
+    flex-direction: column;
+
+    @include media-query(medium) {
+        padding: $grid * 2;
+    }
+
+    @include media-query(large) {
+        padding: 0 $grid * 4 $grid * 4 $grid * 4;
+        flex-direction: row;
+    }
+}
+
+.textContainer {
+    text-align: left;
+    margin-bottom: $grid;
+    order: 1;
+
+    @include media-query(medium) {
+        order: 2;
+    }
+
+    @include media-query(large) {
+        max-width: 635px;
+        margin-bottom: 0;
+    }
+}
+
+.title {
+    @include font-size(xxxxl);
+    font-weight: $weight--extrabold;
+    line-height: 1;
+}
+
+.image {
+    display: none;
+    @include media-query(large) {
+        display: block;
+        order: 1;
+        margin: auto $grid * 2 $grid * 2 auto;
+        transform: rotate(0deg);
```

Comment:
**Suggestion**: I think this can be removed otherwise a comment explaining why it's needed

Reply:
Removed, this was left over from debugging